apps/meteor/client/views/room/contextualBar/RoomMembers/AddUsers/AddUsersWithData.tsx

```diff
@@ -52,13 +65,44 @@ const AddUsersWithData = ({ rid, onClickBack, reload }: AddUsersWithDataProps):
 			dispatchToastMessage({ type: 'error', message: error as Error });
 		}
 	});
+
+	const handleMatrixValidation = useMutableCallback(async () => {
```

Comment:
I think it would make more sense to move this whole function to the `useMatrixIdValidation` hook.